commands/random/index.js

```diff
@@ -1,100 +1,53 @@
 const discord = require("discord.js")
+const {dispHelp} = require("./help");
+const {randInt, randFloat} = require("./randnbr");
+const {randList, randPeople} = require("./randtext");
 
 module.exports = {
     description: 'Bot gives a random number, between the specified bounds if they exist',
-    usage: 'random <type> <params>',
+    usage: 'random <type> [params]',
```

Comment:
*usage* doesn't need the command name

```suggestion
    usage: '<type> [params]',
```